.env.development

```diff
@@ -6,6 +6,3 @@
 # @TRAIN
  VITE_API_URL=https://360ApiTrain.gordon.edu/
 
-# @LOCALHOST
-#  VITE_API_URL=http://localhost:51660/
-
```

Comment:
This needs to be reverted - `.env.development` should have a Localhost section that is commented out by default